ansible_base/lib/utils/create_system_user.py

```diff
@@ -0,0 +1,60 @@
+import logging
+from typing import Optional, Tuple
+
+from django.core.exceptions import ImproperlyConfigured
+from django.utils.translation import gettext as _
+
+from ansible_base.lib.utils.settings import get_setting
+
+logger = logging.getLogger('ansible_base.lib.utils.create_system_user')
+
+"""
+These functions are in its own file because it is loaded during migrations so it has no access to models.
+"""
+
+
+def create_system_user(user_model=None) -> object:  # Note: We can't load models here so we can typecast to anything better than an object
```

Comment:
I'm pretty sure you could say `Model` instead of `object`, but neither are wrong. You might even be able to say the Django `AbstractUser`. Because while the principle is that our utils shouldn't import from "our" models, our utils can import for _Django_ models while preserving the intended import ordering.

Reply:
I tried to make it AbstractUser but that was unable to be imported because we're still loading when calling this from a migration and, from that I can tell, you can't make a type hint variable (to load the model from an app).

Reply:
I can try model.Model, that would at least be more specific than object.

Reply:
`models.Model` it is.